tests/e2e/objects/views/TopicsView.ts

```diff
@@ -45,13 +45,21 @@ export class TopicsView extends View {
   /** Click the "Refresh" nav action in the view title area. */
   async clickRefresh(): Promise<void> {
     await this.clickNavAction("Refresh");
+    // wait for any loading to complete before returning
+    await expect(this.progressIndicator).toBeHidden();
   }
```

Comment:
This may be something added to the other view models, but the Topics view provider is the only one that doesn't behave the same as the others (since it doesn't subclass from `BaseViewProvider`/`ParentedBaseViewProvider`)